subjects/devops/append-output/README.md

```diff
@@ -0,0 +1,61 @@
+## append-output
+
+### Instructions
+
+Create a file `append-output.sh` that will get the output of a file and parse it, and then write it to a file with a specific format using a single command.
+
+Get the content of the `songs.txt`, parse it with the `grep` command to filter the file in order to get all the songs from the artist whose names start with `J`, and write the output to the existing file `results.txt`, check the example below:
```

Comment:
It's still not clear whether the `songs.txt` and `results.txt` files are supposed to be provided, or if they have to be created and submitted by the learner or not.

Reply:
The must submit only the script.
i gave the files to them, together with the subject, for them to test their solution and i've added a message saying that only the script must be submitted